src/apps/app/Routes.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { Switch, Route } from 'react-router-dom'
+import parseRoute from '../../v2/util/parseRoute'
+
+// Feed
+import FeedPage from '../../v2/pages/feed/FeedPage'
+import NotificationPage from '../../v2/pages/feed/NotificationPage'
+// Profile
+import ProfilePage from '../../v2/pages/profile/ProfilePage'
+// Channel
+import ChannelPage from '../../v2/pages/channel/ChannelPage'
+import SharedChannelPage from '../../v2/pages/channel/SharedChannelPage'
+import ChannelFollowersPage from '../../v2/pages/channel/ChannelFollowersPage'
+import EmbeddedChannelPage from '../../v2/pages/channel/EmbeddedChannelPage'
+// Block
+import BlockPage from '../../v2/pages/block'
+
+export const Routes = () => {
+  return (
+    <Switch>
+      {/* Feed */}
+      <Route exact path="/feed" component={FeedPage} />
+
+      <Route exact path="/notifications" component={NotificationPage} />
+
+      <Route exact path="/" component={FeedPage} />
+
+      {/* Block */}
+      <Route
+        exact
+        path="/block/:id"
+        render={parseRoute(({ params }) => (
+          <BlockPage id={parseInt(params.id, 10)} />
+        ))}
+      />
+
+      {/* Profile */}
+      <Route
+        exact
+        path="/:id/:view(all|index|blocks|channels|followers|following|groups)?"
```

Comment:
This is how we get around the ambiguous routing issue here. It'll only match the profile routes if it's constrained by one of these reserved words, otherwise falls through to the channel.